jsk_perception/cfg/ConcaveHullMaskImage.cfg

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env python
+
+PACKAGE='jsk_perception'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add("min_area", double_t, 0, "minimum area of mask region", 0, 0.0, 10000000)
+gen.add("max_area", double_t, 0, "maximum area of mask region", 10000000, 0.0, 10000000)
```

Comment:
Hmm, I thought this parameter can change by users and did not take image size always.
Because of some image has small area of contours and we ignore it.
original image
![left0000](https://cloud.githubusercontent.com/assets/4690682/24504728/4fb70590-1592-11e7-91bc-540ca1a47ab1.jpg)

maximum area of mask region is image size.
![left0000](https://cloud.githubusercontent.com/assets/4690682/24504668/195fa7a4-1592-11e7-8cb5-0580286950a2.jpg)

ignore small contour for area limitation.
![left0046](https://cloud.githubusercontent.com/assets/4690682/24504690/2ee11d2e-1592-11e7-9746-1b01fd7358c2.jpg)